pkg/util/collections/includes_excludes.go

```diff
@@ -101,6 +105,121 @@ func (ie *IncludesExcludes) ShouldInclude(s string) bool {
 	return ie.includes.Len() == 0 || ie.includes.Has("*") || ie.includes.match(s)
 }
 
+// IncludesExcludesInterface is used as polymorphic IncludesExcludes for Global and scope
+// resources Include/Exclude.
+type IncludesExcludesInterface interface {
+	ShouldInclude(s string) bool                        // Checkout whether the passed in parameter name should be included.
```

Comment:
This interface should work fine if it only contain on func `ShouldInclude` 

Additionally we should add more details regarding the input param `s`, like should it be a `resource` or `object`, should it include namespace, should it be fully qualified, etc.

Reply:
Modified.